docs/restricting-api-access.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Restricting API access
+sidebar_position: 2
+---
+
+Even though Saleor [implements complex authorization and authentication](/api-usage/authentication.mdx) mechanisms, you might want to restrict access to your Saleor Cloud instance for specific use cases.
+
+Below, we explore two ways Saleor Cloud allows you to restrict access to the API, as well as a practical example of how to use one of them.
+
+## Allowed API origins
+
+Most times, the list of the origins you expect to call your API is known in advance. By specifying the allowed origins, you can prevent unauthorized access to your API.
+
+In the project view of your Saleor Cloud dashboard, you can customize the allowed origins. The default option is to "Allow all", but you can also specify a list of origins or set it to "Dashboard only".
+
+:::info
+
+Changing the settings to only allow the selected origins can affect the installed apps. If you want your app to call the Saleor API from the client, you need to add the app's origin to the list of allowed origins. You can read more about app's access scopes in the [App permissions](developer/extending/apps/architecture/app-permissions.mdx#access-scopes) document.
+
+:::
+
+Here is how an example list of allowed origins may look:
```

Comment:
> Are wildcards (eg https://*.example.com) supported? If so, it would be good idea to mention it here.

I don't think so, the "*" character isn't even allowed by the regex.